coderd/database/pubsub/pubsub.go

```diff
@@ -239,20 +249,22 @@ func (p *pgPubsub) subscribeQueue(event string, newQ *msgQueue) (cancel func(),
 	}, nil
 }
 
-func (p *pgPubsub) Publish(event string, message []byte) error {
+func (p *PGPubsub) Publish(event string, message []byte) error {
 	p.logger.Debug(p.ctx, "publish", slog.F("event", event), slog.F("message_len", len(message)))
 	// This is safe because we are calling pq.QuoteLiteral. pg_notify doesn't
 	// support the first parameter being a prepared statement.
 	//nolint:gosec
 	_, err := p.db.ExecContext(p.ctx, `select pg_notify(`+pq.QuoteLiteral(event)+`, $1)`, message)
 	if err != nil {
+		p.publishesTotal.WithLabelValues("false").Inc()
```

Comment:
Does false label represent error here? Is that common prom metric practice? (Just checking since I'm not that experienced adding prom metrics.)

Reply:
The label name is `success` with values `true` or `false`.  Just trying to keep our labeling consistent within the product and that's how we do it at https://github.com/coder/coder/blob/3ace7982aabe15269093d570feb36a58f7c2f8e6/agent/agentscripts/agentscripts.go#L66